it/mail/chrome/messenger/accountCreationModel.properties

```diff
@@ -3,11 +3,11 @@
 
 
 # readFromXML.js
-no_emailProvider.error=Il file XML non contiene la configurazione di un account di posta.
+no_emailProvider.error=Il file XML di configurazione non contiene la configurazione dell'account e-mail.
```

Comment:
CORRECTIONS NEEDED. There is an unnecessary repetition ("configurazione"). I suggest to change the sentence as follows: 

 +no_emailProvider.error=Il file XML non contiene la configurazione dell'account e-mail.